xml/System.Security.Cryptography/HMACMD5.xml

```diff
@@ -191,7 +191,7 @@
         <Parameter Name="key" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="key">The secret key for <see cref="T:System.Security.Cryptography.HMACMD5" /> encryption. The key can be any length, but if it is more than 64 bytes long it will be hashed (using SHA-1) to derive a 64-byte key. Therefore, the recommended size of the secret key is 64 bytes.</param>
+        <param name="key">The secret key for <see cref="T:System.Security.Cryptography.HMACMD5" /> encryption. The key can be any length, but if it is more than 64 bytes long it will be hashed (using MD5) to derive a 16-byte key. Therefore, the recommended size of the secret key is 64 bytes.</param>
```

Comment:
While we're editing these, can we change all of the `<see cref=T:THETYPEIMON"/> encryption" to "HMAC computation".  Since MAC != encryption, and the self-reference doesn't seem worthwhile here.

Reply:
Yeah. Good idea.